src/controllers/dishes.controller.ts

```diff
@@ -0,0 +1,50 @@
+import { Request, Response } from 'express';
+import * as dishHandler from '../handlers/dishes.handler';
+
+export const addNewDish = async (req: Request, res: Response) => {
+    try {
+        const newDish = await dishHandler.addNewDish(req.body);
+        res.status(201).json(newDish);
+    } catch (error: any) {
+        res.status(500).json({ message: error.message });
+    }
+};
+
+export const getAllDishes = async (req: Request, res: Response) => {
+    try {
```

Comment:
Same here, you are bringing the dishes that are deleted, please filter them here and in getDishById (in that function you will need to return 400 if the dish was deleted)